flatpak/qt/org.nanuc.Axolotl.desktop

```diff
@@ -0,0 +1,10 @@
+# https://docs.flatpak.org/en/latest/freedesktop-quick-reference.html#desktop-files
+
+[Desktop Entry]
+Name=Axolotl
+Exec=flatpak run org.nanuc.Axolotl -axolotlWebDir=/app/bin/axolotl-web/dist -e qt
```

Comment:
I believe we need an equal sign between the key and the value here: `-e=qt`. Similarly to `-axolotlWebDir=/app/bin/axolotl-web/dist` (which we might not need anymore, but lets keep it just in case)

Reply:
I can confirm that this is no-longer required. I've done all my testing without realizing that it was the standard, but for conformity I'll add it.